dev/tests/integration/testsuite/Magento/Sales/Model/ResourceModel/Order/Customer/CollectionTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Sales\Model\ResourceModel\Order\Customer;
+
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class CollectionTest extends TestCase
+{
+    /**
+     * @var Collection
+     */
+    private $collection;
+
+    /**
+     * @inheritDoc
+     */
+    public function setUp()
+    {
+        $objectManager = Bootstrap::getObjectManager();
+        $this->collection = $objectManager->get(Collection::class);
+    }
+
+    /**
+     * Attribute presence test
+     *
+     * @magentoDataFixture Magento/Customer/_files/customer.php
+     * @magentoDataFixture Magento/Customer/_files/customer_address.php
+     * @dataProvider joinAttribute
+     * @param string $attribute
+     * @return void
+     */
+    public function testAttributePresent($attribute): void
+    {
+        $customers = $this->collection->getItems();
+        foreach ($customers as $customer) {
+            $this->assertNotEmpty($customer->getData($attribute), "Attribute '$attribute' does not present");
```

Comment:
Hi @engcom-Echo. Would you adjust the assertion message, please? It's not a big deal but may hurt the eye

```suggestion
            $this->assertNotEmpty($customer->getData($attribute), "Attribute '$attribute' is not present");
```